### PR TITLE
fix: set minimum width for grid fields to 100px

### PR DESCRIPTION
**Pull request by Nitin-Poojary**
fixes [3388](https://github.com/AppFlowy-IO/AppFlowy/issues/3388)

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

**Comment by Nitin-Poojary** [comment]
@richardshiue Do I need to write tests for this?

**Comment by Nitin-Poojary** [comment]
Okay.

**Comment by Nitin-Poojary** [comment]
Is there a test which checks for resizing of the field width? That would be helpful.

**Comment by Nitin-Poojary** [comment]
Hey, can you help me with this "Invalid argument(s): Failed to load dynamic library 'AppFlowy\frontend\appflowy_flutter/.sandbox/dart_ffi.dll': error code 126". Got this while running bloc tests.

**Comment by Xazin** [comment]
> Hey, can you help me with this "Invalid argument(s): Failed to load dynamic library 'AppFlowy\frontend\appflowy_flutter/.sandbox/dart_ffi.dll': error code 126". It's occurring due to the ensureInitialized() in setUpAll() function.

Can you check if this is your issue, and see if the solution resolves it for you?
https://docs.appflowy.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/troubleshooting#18.-cant-run-tests-on-windows-failed-to-load-dynamic-library

**Comment by richardshiue** [comment]
Hello @Nitin-Poojary, we don't have an existing integration test for field resize.

File: frontend/appflowy_flutter/lib/plugins/database_view/application/field/field_cell_bloc.dart
Comment (richardshiue) [inline frontend/appflowy_flutter/lib/plugins/database_view/application/field/field_cell_bloc.dart:36]: I think 50px would be better here. Suppose I just have a checkbox

This logic works, but is not backwards compatible. Maybe you could think about what would happen if my field width was already less than 100 (or the 50 like I mentioned above) before this patch?

**Review by richardshiue — CHANGES_REQUESTED** [review]

File: frontend/appflowy_flutter/lib/plugins/database_view/application/field/field_cell_bloc.dart
Comment (Nitin-Poojary) [reply frontend/appflowy_flutter/lib/plugins/database_view/application/field/field_cell_bloc.dart]: Is this fine? If a user already has a field with a width less than 50px, it will become 50px when the user tries to change its width.

**Comment by Nitin-Poojary** [comment]
Okay, I also wanted to clarify that. Thanks for clearing it beforehand.

**Review by richardshiue — APPROVED** [review]
LGTM!

**Comment by Nitin-Poojary** [comment]
So, every time I make a push, I have to wait for your approval for the CI workflows to run? How can I fix failing tests like this?

**Comment by richardshiue** [comment]
Hey there @Nitin-Poojary, I understand your frustration. Like Mathias has said, you can and should test locally before pushing to the PR. This is a mechanism to protect abuse on the CI and at the moment only applies to first time contributors. Once your first PR gets merged, future PRs will automatically trigger the CI and won't have to get approved first.

**Comment by Nitin-Poojary** [comment]
I was never frustrated in the first place. The message I sent was not out of frustration; I was genuinely asking. Anyway, even though it was not intentional, if it sounded rude in any way, I apologize for that. On another note, I need some assistance in resolving these CI test failures. The integration test fails and Flutter CI build fails for Windows are both occurring during the building of AppFlowy and working fine locally, and I'm unsure how to fix them. Additionally, every test failure in the integration test for Ubuntu is also failing locally. From what I can gather, every test that contains the tester.createNewPageWithName() function multiple times is failing, and I'm not sure of the reason for that either.

**Comment by richardshiue** [comment]
Ahh I didn't mean it like that. I was just explaining why it exists. The current issue with the integration tests is not a result of this PR but affects every right now. It should get fixed by #3472 . I'm actually fine with straight up merging this PR as is, but it'd be much safer to wait for that to get merged, merge main into your branch, then pass the CI.

**Comment by richardshiue** [comment]
Merged! Thanks for contributing!

**Pull request merged**
